public/app/percona/dbaas/components/DBCluster/DBCluster.tsx

```diff
@@ -119,6 +133,18 @@ export const DBCluster: FC = () => {
     [setSelectedCluster, setDeleteModalVisible, getDBClusters]
   );
 
+  const [initialValues, setInitialValues] = useState<Record<string, any>>(() => {
```

Comment:
I'm looking forward to a lot of conflicts with my (https://github.com/percona-platform/grafana/pull/468) =,)

Reply:
Don't worry, you can merge yours and I'll solve them :)